test/integration/targets/inventory_aws_conformance/runme.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# set the output dir
+if [ -z ${OUTPUT_DIR+null} ]; then
+    export OUTPUT_DIR=$(pwd)
+fi
+
+#################################################
+#   RUN THE SCRIPT
+#################################################
+
+# run the script first
+cat << EOF > $OUTPUT_DIR/ec2.ini
+[ec2]
+regions = us-east-1
+cache_path = $(pwd)/.cache
+cache_max_age = 0
+group_by_tag_none = False
+
+[credentials]
+aws_access_key_id = FOO
+aws_secret_acccess_key = BAR
+EOF
+
+rm -f script.out
+./ec2.sh | tee -a $OUTPUT_DIR/script.out
+#./ec2.sh 
+RC=$?
+if [[ $RC != 0 ]]; then
+    exit $RC
+fi
+#rm -f $OUTPUT_DIR/ec2.ini
+#rm -f $OUTPUT_DIR/ec2.py
+rm -rf .cache
+
+#exit 0
+
+#################################################
+#   RUN THE PLUGIN
+#################################################
+
+# run the plugin second
+export ANSIBLE_INVENTORY_ENABLED=aws_ec2
+export ANSIBLE_INVENTORY=test.aws_ec2.yml
+export AWS_ACCESS_KEY_ID=FOO
+export AWS_SECRET_ACCESS_KEY=BAR
+
+cat << EOF > $OUTPUT_DIR/test.aws_ec2.yml
+plugin: aws_ec2
+cache: False
+regions:
+    - us-east-1
+hostnames:
+  - dns-name
+compose:
+  # vars that don't exist
+  ansible_host: public_dns_name
+  ec2_item: backwardscompat | default("")  # boto anomaly, always appears to be u'\n', don't see a reason to keep it around
+  ec2_monitoring: backwardscompat | default("") # boto anomaly, always appears to be u'\n', don't see a reason to keep it around
```

Comment:
FYI, it is believed that there is a value somewhere in the current `monitoring` dictionary which corresponds to this old value, and the reason we haven't found it yet is because the key will not be present if monitoring isn't turned on... and we have yet to experiment with turning it on to see what pops up.

Reply:
I have turned on monitoring for instances and boto never appears to return an informative value for it, just as a newline containing spaces. If you find out how to access a meaningful value for it via the script (or plugin) at some point please let me know.

Reply:
oh that's further than I went with it, so it's probably a lost cause.